doc/manipulating_visualizing/manipulating_images.rst

```diff
@@ -32,21 +32,25 @@ datasets and atlases. Dataset fetching functions can be imported from
 :mod:`nilearn.datasets`::
 
     >>> from nilearn import datasets
-    >>> haxby_files = datasets.fetch_haxby(n_subjects=1)  # doctest: +SKIP
+    >>> haxby_dataset = datasets.fetch_haxby(n_subjects=1)
```

Comment:
If you call `nilearn.datasets.tests.test_utils.setup_mock`, the function `fetch_haxby` will return existing paths to empty files.